numpy/linalg/tests/test_linalg.py

```diff
@@ -1828,6 +1828,7 @@ def test_xerbla_override():
             pytest.skip('Numpy xerbla not linked in.')
 
 
+@pytest.mark.skip(reason='gh-13478')
```

Comment:
This is unfortunate, but I couldn't manage to get i.e., an environment variable to pass through the SDE for selective skipping in the CI job.

It would also be useful if I could use `runtests.py -s "linalg" -- -k "not test_sdot_bug_8577"`, but I don't think mixing and matching test selection between `runtests.py` and the args fed to `--` (pytest) is fool-proof. 

A further complication is the potential special meaning of `--` after the emulation command, and not sure how well emulation works with pytest-xdist if we want to use many cores...